core/modules/views_ui/js/views-admin.js

```diff
@@ -63,12 +65,21 @@ Backdrop.behaviors.viewsUiAddView = {
     if ($feedField.length) {
       if (!this.feedFiller) {
         suffix = '.xml';
-        this.feedFiller = new Backdrop.viewsUi.FormFieldFiller($feedField, replace, suffix);
+        this.feedFiller = new Backdrop.viewsUi.FormFieldFiller($feedField, replace_dash, suffix);
       }
       else {
         this.feedFiller.rebind($feedField);
       }
     }
+    var $machineNameField = $context.find('[id^="edit-name"]');
+    if ($machineNameField.length) {
+      if (!this.machineNameFiller) {
+        this.machineNameFiller = new Backdrop.viewsUi.FormFieldFiller($machineNameField, replace_underscore);
+      }
+      else {
+        this.machineNameFiller.rebind($machineNameField);
+      }
+    }
   }
```

Comment:
@klonos
This works, but I'm concerned that the `machine_name` is given special treatment on the initial `admin/structrure/views/add` page, which as far as I can tell is to do the RegEx processing for transliteration.

So, the construct here adds the text to the ui the same way as the other elements and I think is bypassing (or never passing to the transliteration).

I'm not sure about any of this so I think we need @quicksketch to chime in.